app/containers/unconnectedAuthorShow/index.tsx

```diff
@@ -315,12 +315,12 @@ class AuthorShow extends React.PureComponent<AuthorShowProps> {
     return (
       <Helmet>
         <title>{author.name}</title>
-        <meta itemProp="name" content={`${author.name} | Scinapse | Academic search engine for paper`} />
+        <meta itemProp="name" content={`${author.name} | Sci-napse | Academic search engine for paper`} />
         <meta name="description" content={description} />
```

Comment:
What I know is that sci-napse is not we want now.

Reply:
wow.. I do not know why I could not identify the element. I corrected the problem.